example_site/src/components/Header.js

```diff
@@ -1,13 +1,20 @@
 import React from 'react'
- 
+import "./Header.css"
+
+/**
+ * Creates a Header reusable component
```

Comment:
What about something like, "Renders a title and subtitle, intended to be used as a header"?